pkg/machines/libvirt-common.es6

```diff
@@ -186,6 +187,17 @@ export function parseDumpxml(dispatch, connectionName, domXml, id_overwrite) {
 
     const currentMemoryUnit = currentMemoryElem.getAttribute("unit");
     const currentMemory = convertToUnit(currentMemoryElem.childNodes[0].nodeValue, currentMemoryUnit, units.KiB);
+    let maxMemoryTmp;
+    if (maxMemoryElem) {
+        const maxMemoryUnit = maxMemoryElem.getAttribute("unit");
+        maxMemoryTmp = convertToUnit(maxMemoryElem.childNodes[0].nodeValue, maxMemoryUnit, units.MiB);
+    } else {
+        const memoryElem = domainElem.getElementsByTagName("memory")[0];
+        const memoryUnit = memoryElem.getAttribute("unit");
+        const memory = convertToUnit(memoryElem.childNodes[0].nodeValue, memoryUnit, units.MiB);
+        maxMemoryTmp = memory;
+    }
+    const maxMemory = maxMemoryTmp;
```

Comment:
maxMemory attribute should not be used for the purpose of this PR, it's for memory hotplugging and we 'll not support this from this PR at least. What you want to use as max memory is always the ```memory``` atttribute.

Reply:
Should I therefore change also names of functions?
i.e.:
SET_MEMORY            ->  SET_CURRENT_MEMORY
SET_MAX_MEMORY  ->  SET_MEMORY
updateMaxMem()         ->  updateMem()

Reply:
whatever you prefer, I don't have strong opinion on this.